src/idiom248ex.fz

```diff
@@ -0,0 +1,10 @@
+ex248 is
+  construct_float(s bool, m f64, e i32) f64 =>
+    (if s then -1.0 else 1.0) * m * (2 ** e).as_f64
+
+
+  test_vals := [(false, 1.0, 10),(false, 13.0, 7),(true, 17.0, 5),(true, 42.0, 8),(true, 0.66, 7),(false, 100.99, 12)]
```

Comment:
Add some LFs and vertical alignment
```suggestion
  test_vals := [(false, 1.0, 10),
                (false, 13.0   , 7),
                (true , 17.0  , 5),
                (true , 42.0  , 8),
                (true , 0.66  , 7),
                (false, 100.99, 12)]
```
Also add some test cases with larger exponents. Does this work close to f64.max?